qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java

```diff
@@ -126,7 +122,7 @@ public void bothReversedTest() throws IOException {
 	}
 	
 	@Test
-	public void toXmlFastqTest() throws ParserConfigurationException {
+	public void toXmlFastqTest() throws Exception {
```

Comment:
are any exceptions thrown from this method? If not, please remove, otherwise be more specific in terms of the type of exception